src/util/heap/layout/map.rs

```diff
@@ -32,6 +32,10 @@ pub trait Map: Sized {
 
     fn get_contiguous_region_size(&self, start: Address) -> usize;
 
+    fn get_available_discontiguous_chunks(&self) -> usize;
```

Comment:
Can you add comments for these methods? https://github.com/JikesRVM/JikesRVM/blob/0b6002e7d746a829d56c90acfc4bb5c560faf634/MMTk/src/org/mmtk/utility/heap/layout/Map.java#L141